packages/eslint-plugin-pf-codemods/src/rules/v6/componentGroupsInvalidObjectPropsRenameToMissingPageProps/component-groups-invalidObjectProps-rename-to-missingPageProps.md

```diff
@@ -0,0 +1,18 @@
+### component-groups-invalidObjectProps-rename-to-missingPageProps [(react-component-groups/#313)](https://github.com/patternfly/react-component-groups/pull/313)
```

Comment:
This isn't a blocker for this PR, but sorta wondering if this should really be its own separate rule or included as part of the pre-existing one (with a rename maybe). Really the question is, "should there be a single rule dealing with anything InvalidObject rename related?"

Doing so would require a little refactoring both to the existing rule and the new helper here possibly, but also depends whether doing so would just bloat the existing rule too much. Having them separate may be more clear and understandable than trying to combine them, but curious what you think.

Reply:
> "should there be a single rule dealing with anything InvalidObject rename related?"
I think ideally there should be just a single rule, but...

> Doing so would require a little refactoring
I am afraid it could end up being "a lot of" refactoring, given that the current `renameComponent` helper already returns a function. But now that I am thinking about it, we could combine it and do something like:
```
create: (context: Rule.RuleContext) => {...renameComponent(context), ...renameInterface(context)}
```
But maybe it will not be that easy